moduleframework/avocado_testers/container_avocado_test.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# This Modularity Testing Framework helps you to write tests for modules
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#
+
+from moduleframework.module_framework import AvocadoTest
+
+
+# INTERFACE CLASSES FOR SPECIFIC MODULE TESTS
+class ContainerAvocadoTest(AvocadoTest):
+    """
+    Class for writing tests specific just for DOCKER
+    derived from AvocadoTest class.
+
+    :avocado: disable
+    """
+
+    def setUp(self):
+        if self.moduleType != "docker":
+            self.skip("Docker specific test")
+        super(ContainerAvocadoTest, self).setUp()
+
+    def checkLabel(self, key, value):
+        """
+        check label of docker image, expect key value (could be read from config file)
```

Comment:
In this PR, lowercase and [uppercase](https://github.com/fedora-modularity/meta-test-family/pull/2/files#diff-d49eca679ceb3a9c6a60a7a4afec5f21R199) are used to start a docstring. Let's agree on uppercase, and dot in the end.